packages/ckeditor5-source-editing/src/sourceediting.js

```diff
@@ -269,14 +282,17 @@ export default class SourceEditing extends Plugin {
 	}
 
 	/**
-	 * Focuses the textarea containing document source from the first editing root.
+	 * Focuses the textarea containing document source from the first editing root. Places the cursor at the beginning of the textarea.
 	 *
 	 * @private
 	 */
 	_focusSourceEditing() {
 		const [ domSourceEditingElementWrapper ] = this._replacedRoots.values();
 
-		domSourceEditingElementWrapper.querySelector( 'textarea' ).focus();
+		const textarea = domSourceEditingElementWrapper.querySelector( 'textarea' );
+
+		textarea.focus();
+		textarea.selectionEnd = 0;
```

Comment:
With this change there's a problem as the textarea can be quite big, the window is scrolled to the bottom, but the selection is placed on top of the textarea.

Reply:
Only in Chrome. In Firefox it works as expected.
But still this should be fixed.

Reply:
I found some hacky ways to prevent Chrome from scrolling to the bottom, but then I decided to remove this behavior at all. This behavior might be caused by an old issue in Chrome, that the text selection or the text cursor is not made visible after the selection is changed from JavaScript code (https://bugs.chromium.org/p/chromium/issues/detail?id=331233).

We have already an issue to map the cursor position from the editing view to the "offset" in the source view (https://github.com/ckeditor/ckeditor5/issues/9797).